docs/sebak_deployment_standalone.md

```diff
@@ -0,0 +1,170 @@
+By the nature of design, SEBAK should be deployed as composition of nodes and they should tightly connected for consensus. This standalone mode recommend only for testing and debugging. With standalone mode, you do not need to compose quorum and another server. It does not make consensus, just execute incoming messages including transactions, but it do the right work, such as inflation, collecting transaction fee, etc.
+
+## Installation
+
+Please follow the [Installation](./sebak_Installation.md). If you are so busy and you already have golang (1.11 or higher) environment, do like this,
```

Comment:
File name link(sebak_Installation.md) should be also modified.

Reply:
I changed it.